### PR TITLE
Rectifying content changes/suggestions

### PR DESCRIPTION
**Pull request by jseppi**
from those made in #1373 and #1344.

Still to do (or decide not to do):
- [x] @wslack - modifications to home page, particularly those around the SVGs (https://github.com/18F/federalist/pull/1373/files#diff-d2d32307738e08558710ff29b2a05c7f)
- [x] README setup instructions (https://github.com/18F/federalist/pull/1344/files#diff-04c6e90faac2675aa89e2176d2eec7d8R35)

### CODE REVIEW

**Comment by wslack** [comment]
I am :man_shrugging: about the SVG updates

**Comment by wslack** [comment]
@jseppi did my updates, edited content but left sections in place

**Comment by jseppi** [comment]
Alright, this looks good to me! @wslack and/or @el-mapache, please give it a final review.

File: README.md [hunk 1]
```diff
@@ -5,11 +5,25 @@
...
 
-***Under active development. Everything is subject to change. Learn more at the [documentation site](https://federalist-docs.18f.gov/). Interested in talking to us? [Join our public chat room](https://chat.18f.gov/).***
+***Federalist is updated regularly. [Join our public chat room](https://chat.18f.gov/) to talk to us and stay informed. You can also check out our [documentation](https://federalist-docs.18f.gov/) to learn more.***
 
-Federalist is a unified interface for publishing static government websites. It automates common tasks for integrating GitHub and Amazon Web Services to provide a simple way for developers to launch new static websites or more easily manage existing static websites. This repo is home to "federalist-core" - a Node.js app that allows government users to add and configure their Federalist sites.
+## About Federalist
 
-## How to set up a local Federalist development server
+[Federalist](https://federalist.18f.gov) helps federal government entities publish compliant static websites quickly and seamlessly. Federalist is built on top of [cloud.gov](cloud.gov), GitHub, and Amazon Web Services.
```
Comment (wslack) [inline README.md]: should we be a little more precise that Federalist integrates with Github?

Comment (jseppi) [reply README.md]: I think this is ok, but what about something like

"Federalist is built on top of cloud.gov and Amazon Web Services, and integrates with GitHub."

Comment (wslack) [reply README.md]: switched it around a bit to clarify that cloud.gov uses AWS and we get AWS via cloud.gov, otherwise is 👍

Comment (jseppi) [reply README.md]: nice

**Pull request merged**